### ISSUE TITLE
I cant get this to run for some reason :/ i tried everything i could and cant find any errors

### ISSUE THREAD

**Issue by Hexfold96** [aseprite/aseprite#2186]
cant get it to run, it just tries to open, but it doesnt.

**Comment by CoderCrane** [comment]
I have the same issue, if a solution is found please let me know.

**Comment by lberrymage** [comment]
Which operating system is this on? What won't run? Please elaborate and give any relevant output you find. I'll be happy to help.

**Comment by Hexfold96** [comment]
I run on the most recent mac update. When you open anything on mac, its shown on the hotbar, and this pops up on the hotbar, starts to open, the crashes completely.

**Comment by lberrymage** [comment]
Have you tried running it from the terminal? See if `open -a /path/to/aseprite.app` does anything. It might give some useful output. You might be able to do `open -a aseprite` or `open -a Aseprite` as well; I don't know as I don't have access to a Mac.

**Comment by CoderCrane** [comment]
I've just followed all of the tutorial steps today and upon getting the aseprite applicaiton within C:\Aseprite\build\bin I've clicked on it to launch and nothing will happen.  The cursor may show the blue circle for a moment but then nothing will happen.  Also, the command prompt states: " 'open' is not recognized as an internal or external command, operable program or batch file " when trying your suggestions.

**Comment by lberrymage** [comment]
`open` is only a command on macOS. On Windows, you should try something along the lines of
`cd C:\Aseprite\build\bin`, then `./aseprite.exe`, substituting "aseprite.exe" if the file name is different.

**Comment by CoderCrane** [comment]
My mistake, I've went through and located the file through command prompt and tried to run it.  This is the result:

 Directory of C:\Aseprite\build\bin

10/21/2019  03:09 PM    <DIR>          .
10/21/2019  03:09 PM    <DIR>          ..
10/21/2019  02:48 PM        10,636,288 aseprite.exe
10/21/2019  02:48 PM        85,471,232 aseprite.pdb
10/21/2019  02:47 PM    <DIR>          data
10/21/2019  02:47 PM         1,130,496 gen.exe
10/21/2019  02:47 PM        12,390,400 gen.pdb
10/21/2019  02:47 PM           564,736 modp_b64_gen.exe
10/21/2019  02:47 PM         2,783,800 modp_b64_gen.ilk
10/21/2019  02:47 PM         5,984,256 modp_b64_gen.pdb
               7 File(s)    118,961,208 bytes
               3 Dir(s)  239,559,475,200 bytes free

C:\Aseprite\build\bin>aseprite.exe

C:\Aseprite\build\bin>Unable to create a user-interface display.
Details: Unknown error

**Comment by lberrymage** [comment]
Which build flags did _you_ use when running CMake?

**Comment by CoderCrane** [comment]
My apologies, as I'm new to coding.  What are build flags?  I've followed the steps listed for windows from the install.md page from start to finish.

**Comment by lberrymage** [comment]
No worries. Flags are options you're passing to a program, so in the case
`cmake -DLAF_OS_BACKEND=skia`, `DLAF_OS_BACKEND=skia` is a flag. It tells the program what options to give to the program to use. I was referring to the ones you passed to cmake. Based on what you're saying, I'm assuming you used the right ones since you don't seem to have deviated from the guide. Did you follow the part about [compiling skia](https://github.com/aseprite/aseprite/blob/master/INSTALL.md#skia-on-windows) before compiling the [whole thing](https://github.com/aseprite/aseprite/blob/master/INSTALL.md#windows-details)?

**Comment by CoderCrane** [comment]
That may be the issue, it mentions compiling skia with clang and I don't think I did that.  It says: "Run GN to generate your build files".  What does that mean?

**Comment by lberrymage** [comment]
You might not've compiled it with clang, but do you know if you compiled it at all? Also you were correct, it is optional to compile with clang, but it's highly recommended to increase the performance of skia.

Did you follow the instructions in [Skia on Windows](https://github.com/aseprite/aseprite/blob/master/INSTALL.md#skia-on-windows) under the section with this text?

>If you haven't installed Clang, and want to compile Skia with MSVC (anyway it's not recommended because the performance penalty is too big), you can use the following commands instead:


Lastly, (I know this is a long comment), can you send a screenshot of where is says "Run GN to generate your build files"? To answer your question, GN is a program to create files that tell the `ninja` compiler how to make the program, which is what Aseprite uses.

**Comment by CoderCrane** [comment]
I've solved my error thanks to your help.  I overlooked: "Here cmake needs different options depending on your platform. You must check the details for Windows, macOS, and Linux. Some cmake options can be modified using tools like ccmake or cmake-gui".  I clicked on the windows link and did an extra step for cmake.  Regarding the screenshot, it is the beginning of the optional compiling with clang.  Thank you so much for your patience, knowledge, and support!
![clang](https://user-images.githubusercontent.com/56849816/67257675-f3b09b80-f441-11e9-9904-7dc71dda5327.PNG)
